tests/unit/test_pytest_mh.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import yaml
+
+from mrack.outputs.pytest_mh import PytestMhOutput
+
+from .mock_data import get_db_from_metadata, provisioning_config
+
+
+@pytest.fixture
+def mock_metadata():
+    return yaml.safe_load(
+        """
+    domains:
+    - name: test
+      type: sssd
+      hosts:
+      - name: dns.test
+        group: medium
+        os: fedora-37
+      - name: client.test
+        group: medium
+        groups:
+        - base_ground
+        - base_client
+        - client
+        role: client
+        os: fedora-37
+        pytest_mh:
+          artifacts:
+          - /etc/sssd/*
+          - /var/log/sssd/*
+          - /var/lib/sss/db/*
+      - name: master.ipa.test
+        group: ipa
+        groups:
+        - base_ground
+        - base_ldap
+        - base_ipa
+        - ipa
+        role: ipa
+        os: fedora-37
+        pytest_mh:
+          config:
+            client:
+              ipa_domain: ipa.test
+              krb5_keytab: /enrollment/ipa.keytab
+              ldap_krb5_keytab: /enrollment/ipa.keytab
+    """
+    )
+
+
+class TestPytestMhOutput:
+    def test_output(self, mock_metadata):
+        config = provisioning_config()
+        db = get_db_from_metadata(mock_metadata)
+
+        mhcfg_output = PytestMhOutput(config, db, mock_metadata)
+        mhcfg = mhcfg_output.create_mh_config()
+
+        expected = yaml.safe_load(
+            """
+        domains:
+        - type: sssd
+          hosts:
+          - hostname: client.test
+            os: linux
+            role: client
+            ssh:
+              host: _gateway
```

Comment:
Any reason to be `_gateway` ? In my local setting the test works if both`_gateway` are replaced by `192.168.0.1` - created in `create_db_host` method. This should solve the CI issue.

```
diff --git a/tests/unit/test_pytest_mh.py b/tests/unit/test_pytest_mh.py
index 39e3e3e..89a60d9 100644
--- a/tests/unit/test_pytest_mh.py
+++ b/tests/unit/test_pytest_mh.py
@@ -66,7 +66,7 @@ class TestPytestMhOutput:
             os: linux
             role: client
             ssh:
-              host: _gateway
+              host: 192.168.0.1
             artifacts:
             - /etc/sssd/*
             - /var/log/sssd/*
@@ -75,7 +75,7 @@ class TestPytestMhOutput:
             os: linux
             role: ipa
             ssh:
-              host: _gateway
+              host: 192.168.0.1
             config:
               client:
                 ipa_domain: ipa.test
(.env) pvoborni@ntb4 ~/d/n/mrack ((76c20d15))> .env/bin/pytest tests/unit/test_pytest_mh.py
========================================= test session starts ==========================================
platform linux -- Python 3.11.1, pytest-7.2.1, pluggy-1.0.0
rootdir: /home/pvoborni/dev/neoave/mrack, configfile: pytest.ini
plugins: asyncio-0.20.3
asyncio: mode=Mode.STRICT
collected 1 item                                                                                       

tests/unit/test_pytest_mh.py .                                                                   [100%]

========================================== 1 passed in 0.34s ===========================================
```

Reply:
_gateway is present in my local test, if I provide the IP address the test fails. But it works in PRCI. Fixed.